<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisTradeCultForm</class>
 <widget class="QMainWindow" name="AxisTradeCultForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AxisTradeCultForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>951</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabOverview">
     <attribute name="title">
      <string>Overview</string>
     </attribute>
     <widget class="QDateEdit" name="DisplayDate">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>110</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="StockGroupsComboBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdateButton">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="UpdateProgressBar">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="SymbolLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Symbol</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="OpenLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="HightLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>High</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="LowLabel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Low</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="CloseLabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="VolumeLabel">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>50</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="ChangeCLabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="ChangeVLabel">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>50</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="AvgC3MLabel">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Avg(3M)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="AvgV3MLabel">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>50</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Avg(3M)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="GraphSampleButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>53</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Graph</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>791</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="StrikePrice1YLabel">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>50</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Strike Price(1Y)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="Stockline">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="AddButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="StockCheckBox">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>50</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="DelButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="OverviewScrollArea">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>90</y>
        <width>931</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(240,240,240)</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>929</width>
         <height>339</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionDataManager"/>
   </widget>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDataManager">
   <property name="text">
    <string>DataManager</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
